moment-range/index.d.ts

```diff
@@ -1,46 +1,57 @@
-// Type definitions for Moment.js 2.0.4
+// Type definitions for Moment-range.js 3.0.3
 // Project: https://github.com/gf3/moment-range
-// Definitions by: Bart van den Burg <https://github.com/Burgov>, Wilgert Velinga <https://github.com/wilgert>
+// Definitions by: Bart van den Burg <https://github.com/Burgov>, Wilgert Velinga <https://github.com/wilgert>, Juan Francisco Adame <https://github.com/franjuan>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import * as moment from 'moment';
 
 export = moment;
 
+type interval = 'years' | 'quarters' | 'months' | 'weeks' | 'days' | 'hours' | 'minutes' | 'seconds' | 'milliseconds';
+
 declare module 'moment' {
      interface Range {
         start: Moment;
         end: Moment;
 
-        contains (other: Date, exclusive?: boolean): boolean;
-        contains (other: Moment, exclusive?: boolean): boolean;
+        contains (other: Range, options?: {exclusive?: boolean}): boolean;
```

Comment:
assuming [this declaration](https://github.com/gf3/moment-range/blob/b4d937a0bd5298e4570f547aa5158f76794c2519/declarations/moment-range.js.flow#L30) is correct, this function should accept all three types, `Date | Moment | Range`